### PR TITLE
solaar: fix g-i and double wrapping

### LINKED ISSUE #185291: Can't run "solaar" after some recent changes
### Describe the bug
If I try to run Solaar via Rofi nothing happens, when trying it via terminal the following error occurred:
> Solaar: missing required system package gir1.2-gtk-3.0

If I open a nix-shell with `nix-shell -p solaar` and run solaar it opens and works without issues.

### Steps To Reproduce
Relevant configuration:
```
hardware.logitech.wireless = {
  enable = true;
  enableGraphical = true;
};
```

1. Rebuild system from unstable
2. Try to run solaar

### Expected behavior
Solaar GUI main windows opens

### Additional context
I've tried to find out more and was directed from @SuperSandro2000 that it could be related to missing `gtk3-x11` but regardless where I added it it didn't solve the issue.

### Notify maintainers
@spinus @ysndr @oxalica

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.15, NixOS, 22.11 (Raccoon), 22.11pre-git`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.10.3`
 - channels(sebtm): `""`
 - channels(root): `""`
 - nixpkgs: `/run/nixpkgs`
```

nixos-unstable -> 168d1c578909dc143ba52dbed661c36e76b12b36

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Description of changes

Closes #185291

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/solaar/default.nix [hunk 1]
```diff
@@ -47,6 +54,17 @@ python3Packages.buildPythonApplication rec {
...
   '';
 
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  # no tests
+  doCheck = false;
```
Comment (oxalica) [inline pkgs/applications/misc/solaar/default.nix:64]: Doesn't it simply do nothing if there are no tests?

**Review by oxalica — COMMENTED** [review]
Builds and works fine.

---

Result of `nixpkgs-review pr 185310` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>logitech-udev-rules</li>
    <li>solaar</li>
  </ul>
</details>

File: pkgs/applications/misc/solaar/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/misc/solaar/default.nix:64]: yes but we document it explicitly otherwise it is unclear in review

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Pull request merged**